bravado/http_client.py

```diff
@@ -197,6 +197,9 @@ def __init__(self, session, request):
         self.session = session
         self.request = request
 
+    def result(self, *args, **kwargs):
```

Comment:
@dnephin can you explain a bit on which part of the implementation is mutable here. I'm kinda lost about it.